### PR TITLE
PLAT-60: Change dashboard addOn manager link to an anchor tag

### PR DESCRIPTION
**Pull request by dondrzzy**
## JIRA TICKET NAME
[PLAT-60: Change dashboard addOn manager link to an anchor tag](https://issues.openmrs.org/browse/PLAT-60)

## Description of what I changed

- Changed the dashboard component addOn manager link to an anchor tag.

## Issue I worked on
[PLAT-60](https://issues.openmrs.org/browse/PLAT-60)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by muhozi — APPROVED** [review]
LGTM

File: app/js/components/Dashboard.jsx [hunk 1]
```diff
@@ -30,8 +31,8 @@ const Dashboard = ({ authenticated }) => {
               <ul className="platform-ul">
                 <div>
                   <li className="platform-list">
-                    <Link
-                      to={addonManagerLink}
+                    <a
+                      href={addonManagerLink}
```
Comment (alexmochu) [inline app/js/components/Dashboard.jsx]: Why did you choose to replace Link with 'a'??

**Review by alexmochu — APPROVED** [review]

File: app/js/components/Dashboard.jsx (hunk 1 above)
Comment (alexmochu) [inline app/js/components/Dashboard.jsx:35]: Why did you choose to replace Link with 'a' ?

Comment (Efosaok) [reply app/js/components/Dashboard.jsx:35]: please respond to above comment @dondrzzy

Comment (dondrzzy) [reply app/js/components/Dashboard.jsx:35]: @Efosaok  and @alexmochu, this change was supposed to be in a previous PR but accidentally left the link in there while updating the PR. But it is explained here: https://github.com/openmrs/openmrs-owa-platformui/pull/12#discussion_r213089260.

Comment (dondrzzy) [reply app/js/components/Dashboard.jsx:35]: I also uploaded the video of what happens when you use the `Link` and `a` tag. Visit the ticket.

**Review by Efosaok — APPROVED** [review]
LGTM

**Pull request merged**